content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -105,10 +105,41 @@ Some Pods have {{< glossary_tooltip text="init containers" term_id="init-contain
 Pods natively provide two kinds of shared resources for their constituent containers:
 [networking](#pod-networking) and [storage](#pod-storage).
 
+### Storage in Pods {#pod-storage}
+
```

Comment:
Is this relocation an improvement? If not, I am happy to revert this change.

Reply:
To be honest, I fail to see the improvement. In the original flow, I can see that both storage and network are used to facilitate communication. Not a strong opinion here though.

Reply:
I initially made this change because the original text and heading seemed more closely related to the content found in the
"Using Pods" section (Pods natively provide two kinds of shared resources for their constituent containers: networking and storage).
From https://kubernetes.io/docs/concepts/workloads/pods/#resource-sharing-and-communication,
`Pods enable data sharing and communication among their constituent containers.`

Reply:
Yes. That is why we have the Pod abstraction. I'm okay with the reorg, though the improvement is not evident to me.
As a non-native speaker, I'm not sure I understand the difference between "Using Pods" and "Working with Pods". So probably this page needs more work in the future.

Reply:
Yes, I agree that the headings, "Using Pods" and "Working with Pods", are similar.

Reply:
I'll look again at the page.